doc/doc.md

```diff
@@ -1229,7 +1229,7 @@ e.g. Markdown
   - `custom_display_name_handler(item, default_handler)` function that formats an item's name. The arguments are the item
 and the default function used to format the name. For example, to show an icon or label beside any
 function tagged with a certain tag:
-
+<code>
```

Comment:
This block is indented with four spaces, which in Markdown parlance should be a code block anyway. Please remove the `<code>` tags here. If needed maybe a blank line before and after the block to separate it from the content.

Reply:
Done.